src/redprl/redprl.grm

```diff
@@ -970,7 +970,7 @@ declArgument
 
 declArguments
   : declArgument ([declArgument])
-  | declArgument SEMI declArguments (declArgument :: declArguments)
+  | declArgument COMMA declArguments (declArgument :: declArguments)
```

Comment:
good, I think this is less confusing anyway.

At some point I'd like to come up with a notation for declarations that looks more like what the "redprl sexprs" look like, but I haven't been able to come up with anything good yet.